packages/graphback-datasync/src/providers/DeltaProvider.ts

```diff
@@ -0,0 +1,203 @@
+import { applyIndexes } from "@graphback/runtime-mongo";
```

Comment:
Name probably wrong - it suggest it is provider. 
It is more like set of the delta table helpers etc.

Reply:
It will change, this whole section is work in progress, I ported this from previous PR about delta tables so it is not completely concrete yet